pyabc/smc.py

```diff
@@ -225,6 +225,14 @@ def __init__(self,
         self.min_acceptance_rate = None
 
         self._sanity_check()
+        self.aknowledge_n_procs()
+
+    def aknowledge_n_procs(self):
+        # if sampler is MulticoreEvalParallelSampler, 
+        # then print number of cores
+        if type(self.sampler) is MulticoreEvalParallelSampler:
+            logger.info(f"Total number of used cores is: 
```

Comment:
pyABC sometimes uses less cores (and sometimes more, if each simulation is parallelized). Maybe: ``f"Parallelizing the sampling on {self.n_procs} cores."``?